test/test_image.py

```diff
@@ -0,0 +1,50 @@
+import os
+import unittest
+
+import torch
+from PIL import Image
+from torchvision.io.image import read_png, decode_png, decode_jpeg, read_jpeg
+import numpy as np
+
+IMAGE_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), "assets", "fakedata", "imagefolder")
+
+
+def get_images(directory, img_ext):
+    assert os.path.isdir(directory)
+    for root, dir, files in os.walk(directory):
+        for fl in files:
+            _, ext = os.path.splitext(fl)
+            if ext == img_ext:
+                yield os.path.join(root, fl)
+
+
+class ImageTester(unittest.TestCase):
+    def test_read_png(self):
+        for img_path in get_images(IMAGE_DIR, "png"):
+            img_pil = torch.from_numpy(np.array(Image.open(img_path)))
+            img_lpng = read_png(img_path)
+            self.assertTrue(torch.all(img_lpng == img_pil))
```

Comment:
It might be better to use [assetEqual](https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_utils.py#L795) since it checks for matching dtypes, etc. etc. ; unless you have a specific reason not to?

Reply:
Good Catch Thanks

Reply:
Good Catch Thanks